test/extended/oauth/client_secret.go

```diff
@@ -62,7 +65,7 @@ var _ = g.Describe("[sig-auth][Feature:OAuthServer]", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 			oc.AddResourceToDelete(userv1.GroupVersion.WithResource("users"), user)
 
-			oauthRoute, err := oc.AdminRouteClient().RouteV1().Routes("openshift-authentication").Get(ctx, "oauth-openshift", metav1.GetOptions{})
+			oauthIssuerURL, err := getOAuthEndpoint(oc.AdminConfig().Host)
```

Comment:
please feel free to ignore, but it seems that what you really want is the token_endpoint, since that's what ends up being used in the request functions ... like https://github.com/openshift/origin/pull/26555/files#diff-6548703e85e00efd742cc59e67cea251a6501651f05151ec0892e02b915dfdc6L117

Reply:
True, but I wanted to keep the change minimal and that would require changing all of the request functions. I don't really have much feelings one way or another though.

Reply:
It does reduce the assumptions about the request path being what it is, but yeah, not a big deal